packages/account/src/Components/poi/status/unsupported/card-details/card-details.scss

```
@@ -192,9 +201,19 @@
         .dc-btn:not(:last-of-type) {
             margin-right: 0.8rem;
         }
-        @include mobile-or-tablet-screen {
+
+        @include tablet-screen {
             height: auto;
             padding-bottom: unset;
+            width: 568px;
+            .dc-btn {
+                width: 50%;
+            }
+        }
+        @include mobile-screen {
+            height: auto;
+            padding-bottom: unset;
+            width: 100%;
 
             .dc-btn {
                 width: 50%;
```

```suggestion
        @include tablet-or-mobile-screen {
    height: auto;
    padding-bottom: unset;
    
    .dc-btn {
        width: 50%;
    }
}

@include tablet-screen {
    width: 568px;
}

@include mobile-screen {
    width: 100%;
}
```


I guess this will work fine as well. 
In this suggestion I have removed repetitive css .

---

works 👍